package.json

```diff
@@ -25,18 +27,23 @@
   },
   "homepage": "https://github.com/Andarist/use-value-callbag#readme",
   "peerDependencies": {
-    "react": "^16.7.0-alpha.0"
+    "react": "^16.8 || ^17"
   },
   "devDependencies": {
     "@babel/core": "^7.1.2",
     "@babel/plugin-transform-modules-commonjs": "^7.1.0",
-    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-env": "^7.13.10",
+    "@babel/preset-react": "^7.12.13",
+    "@testing-library/react-hooks": "^5.1.0",
+    "callbag": "^1.4.0",
+    "callbag-common": "^0.1.8",
```

Comment:
could we replace that with `callbag-subscribe` & `callbag-pipe`? it would be slightly better for me as it would be easier to track which versions of those dependencies this is using

Reply:
yes sure, I just added it there for convenience of future tests 😅😅